azerothcore/azerothcore-wotlk
# fix(DB/SSC): update timers of trash and boss adds in Serpentshrine Cavern with sniffs

## Changes Proposed:
This PR proposes changes to:

## Issues Addressed:

Closes unreported issue

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:

untested by me as of yet

## How to Test the Changes:



Do a general run of SSC on the PTR/local AC to check if the timers do make some sort of sense

## Known Issues and TODO List:


- [ ] fix timers of the bosses
- [ ] Serpentshrine Sporebat unchanged due to open PRs
- [ ] some timers I didn't change because the timers were unreliable (usually due to combat length)


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
